firebase-firestore/src/main/java/com/google/firebase/firestore/local/LruGarbageCollector.java

```diff
@@ -91,4 +157,79 @@ int removeTargets(long upperBound, SparseArray<?> activeTargetIds) {
   int removeOrphanedDocuments(long upperBound) {
     return delegate.removeOrphanedDocuments(upperBound);
   }
+
+  Results collect(SparseArray<?> activeTargetIds) {
+    if (params.minBytesThreshold == Params.COLLECTION_DISABLED) {
+      Logger.debug("LruGarbageCollector", "Garbage collection skipped; disabled");
+      return Results.DidNotRun();
```

Comment:
This should go away as well if you do not instantiate the `LruGarbageCollector` if GC is disabled.